Sat Oct 11 20:10:38 SAST 2025 : Apache Derby Network Server - 10.17.1.0 - (1913217) started and ready to accept connections on port 1527
Sat Oct 11 20:10:39 SAST 2025 Thread[#29,DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3208)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:767)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:437)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:629)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:295)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:921)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1474)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1424)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3378)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1208)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1008)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:298)
Caused by: ERROR XJ004: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:299)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 17 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:299)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3208)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:767)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:437)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:629)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:295)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:921)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1474)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1424)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3378)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1208)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1008)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:298)
============= end nested exception, level (1) ===========
Cleanup action completed
Sat Oct 11 20:10:39 SAST 2025 Thread[#29,DRDAConnThread_2,5,main] (DATABASE = HospitalManagementSystem), (DRDAID = {1}), Database 'HospitalManagementSystem' not found.
Sat Oct 11 20:10:42 SAST 2025 Thread[#29,DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3208)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:767)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:437)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:629)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:295)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:921)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1474)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1424)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3378)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1208)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1008)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:298)
Caused by: ERROR XJ004: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:299)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 17 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:299)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3208)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:767)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:437)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:629)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:295)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:921)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1474)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1424)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3378)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1208)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1008)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:298)
============= end nested exception, level (1) ===========
Cleanup action completed
Sat Oct 11 20:10:42 SAST 2025 Thread[#29,DRDAConnThread_2,5,main] (DATABASE = HospitalManagementSystem), (DRDAID = {2}), Database 'HospitalManagementSystem' not found.
Sat Oct 11 20:10:50 SAST 2025 Thread[#29,DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3208)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:767)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:437)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:629)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:295)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:921)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1474)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1424)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3378)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1208)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1008)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:298)
Caused by: ERROR XJ004: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:299)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 17 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'HospitalManagementSystem' not found.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:299)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3208)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:767)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:437)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:629)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:295)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:921)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1474)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1424)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3378)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1208)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1008)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:298)
============= end nested exception, level (1) ===========
Cleanup action completed
Sat Oct 11 20:10:50 SAST 2025 Thread[#29,DRDAConnThread_2,5,main] (DATABASE = HospitalManagementSystem), (DRDAID = {3}), Database 'HospitalManagementSystem' not found.
----------------------------------------------------------------
Sat Oct 11 20:11:08 SAST 2025: Shutting down Derby engine
----------------------------------------------------------------
